modules/geo-layers/src/tile-layer/tileset-2d.js

```diff
@@ -273,7 +279,7 @@ export default class Tileset2D {
         onTileError: this.onTileError
       });
       Object.assign(tile, this.getTileMetadata(tile));
-      tile.loadData(this._getTileData);
+      tile.loadData(this._getTileData, this._requestScheduler);
```

Comment:
`loadData` also needs to be called on tiles with canceled requests

Reply:
I think I was able to fix this by adding a `tile.isCancelled` attribute, and calling `tile.loadData` again if it was true.